pkg/workers/reconciler_config.go

```diff
@@ -0,0 +1,32 @@
+package workers
+
+import (
+	"time"
+
+	"github.com/spf13/pflag"
+)
+
+type ReconcilerConfig struct {
+	ReconcilerRepeatInterval               time.Duration `json:"reconciler-repeat-interval"`
+	LeaderLeaseExpirationTime              time.Duration `json:"leader-lease-expiration-time"`
+	LeaderElectionReconcilerRepeatInterval time.Duration `json:"leader-election-reconciler-repeat-interval"`
```

Comment:
```suggestion
	ReconcilerRepeatInterval               time.Duration `json:"reconciler_repeat_interval"`
	LeaderLeaseExpirationTime              time.Duration `json:"leader_lease_expiration_time"`
	LeaderElectionReconcilerRepeatInterval time.Duration `json:"leader_election_reconciler_repeat_interval"`
```
json annotation name changed to make it consistency with what we have.
Flags are following the `kebab-case` and the json annotation are following the `snake_case`